### DISCUSSION TITLE
feedback widget not displaying

### DISCUSSION THREAD

**Discussion by poojapm01** [squidfunk/mkdocs-material#5763] [Q&A]
I am integrating feedback widget for my project's documentation site. Configuration for google analytics (data connection successful), consent and feedback widget are done on mkdocs.yml file. Now when I try to publish the site to gh-pages, the feedback widget is not displaying although it's working fine when run locally. How can I resolve this issue?

![image](https://github.com/squidfunk/mkdocs-material/assets/117080977/76ef971b-96bb-487a-bb58-3e86180fb6c1)

Please provide the required help.
Thank you.

**Comment by squidfunk** [comment]
Please provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), as otherwise we can only guess what might be wrong.

**Reply by poojapm01** [reply]
Figured it out in the process, bug was related to versioning. Now feedback widget is working fine. Thank you for your response.
